### PR TITLE
Update specter docs

### PR DESCRIPTION
**Pull request by BitcoinMechanic**

### CODE REVIEW

File: site/source/user-manual/service-guides/specter/specter-service.rst [hunk 1]
```diff
@@ -11,184 +11,253 @@ Electrs will be more performant but will use more system resources.
...
+            .. figure:: /_static/images/services/specter/cc-complete.png
                 :width: 60%
-                :alt: img-9
+                :alt: cc-complete
 
-        #. You may be asked for your PIN and passphrase here, and you may need to confirm extraction on the signing device.
+    .. group-tab:: USB Connected Signing Devices:
 
-        #. Specter will then begin extracting the public key(s) from your device:
+        For devices like the Trezor One on which air gapping is not possible xpubs must be imported via USB - Specter permits this via the HWI which requires running a second instance of Specter on your *local* machine (i.e *not* your server).
```
Comment (kn0wmad) [inline site/source/user-manual/service-guides/specter/specter-service.rst:106]:
```suggestion
        For devices that do not support air-gap features, such as the Trezor One, xpubs must be imported via USB - Specter permits this via the HWI which requires running a second instance of Specter on your *local* machine (i.e *not* your server).
```

**Review by kn0wmad — CHANGES_REQUESTED** [review]
Would be nice to have a quick explainer on why you would even want to use Specter from your server at all, especially for those with USB devices that have to install to their client devices regardless.

**Comment by BitcoinMechanic** [comment]
> Would be nice to have a quick explainer on why you would even want to use Specter from your server at all, especially for those with USB devices that have to install to their client devices regardless.

Done. Still got more work to do on Desktop guide. Will have done by end of day.

File: site/source/user-manual/service-guides/specter/specter-service.rst [hunk 2]
```diff
@@ -1,14 +1,17 @@
 .. _specter-service:
 
-Specter Service
----------------
+====================
+Specter as a Service
```
Comment (kn0wmad) [inline site/source/user-manual/service-guides/specter/specter-service.rst:4]:
```suggestion
Specter Server
```

Let's avoid "as a service" - it has connotations

**Review by kn0wmad — CHANGES_REQUESTED** [review]

File: site/source/user-manual/service-guides/specter/specter-service.rst (hunk 2 above)
Comment (BitcoinMechanic) [reply site/source/user-manual/service-guides/specter/specter-service.rst:4]: I agree, the tradeoff here though is that I have to be very clear about when I'm referring to Specter running on the server vs Specter Desktop. It's a confusing setup - I'm going to research if the HWI can run as a daemon on the client device without using Specter Desktop now.

**Comment by kn0wmad** [comment]
Specter is no longer officially supported

**Pull request closed without merging**